src/sentry/digests/backends/dummy.py

```diff
@@ -1,22 +1,36 @@
 from contextlib import contextmanager
+from typing import TYPE_CHECKING, Any, Iterable, Optional
 
 from sentry.digests.backends.base import Backend
 
+if TYPE_CHECKING:
+    from sentry.digests import Record, ScheduleEntry
+    from sentry.models import Project
+
 
 class DummyBackend(Backend):
-    def add(self, key, record, increment_delay=None, maximum_delay=None):
+    def add(
+        self,
+        key: str,
+        record: "Record",
+        increment_delay: Optional[int] = None,
+        maximum_delay: Optional[int] = None,
+        timestamp: Optional[float] = None,
+    ) -> bool:
         pass
 
-    def enabled(self, project):
+    def enabled(self, project: "Project") -> bool:
         return False
 
     @contextmanager
-    def digest(self, key, minimum_delay=None):
+    def digest(self, key: str, minimum_delay: Optional[int] = None) -> Any:
         yield []
 
-    def schedule(self, deadline):
-        return
-        yield  # make this a generator
+    def schedule(
+        self, deadline: float, timestamp: Optional[float] = None
+    ) -> Iterable["ScheduleEntry"]:
+        return []
+        # yield  # TODO(mgaeta): make this a generator
```

Comment:
Are you still working on this?

Reply:
Yeah I'll be updating this function in another PR.

Reply:
In that case I'd just leave the original implementation alone (you can add type:ignore to avoid type errors if required). I think it's better to not change the implementation in this PR.